### PR TITLE
Adding GCI Mentor Section

### LINKED ISSUE #7: Create a mentors section
We just added a new template
Create a mentors section in the current template with the same design.

The goal would be to create a mentor section, add a mentors.yml file, define a standard format, and make a mentor appear beautifully in the site using Jekyll.

_note: be sure to ask mentors before posting any of their details_

Ref: https://GitHub.com/fossasia/gci17.fossasia.org

### PR DESCRIPTION
**Pull request by sahildav**
Fixes #7 
-- New Changes can be viewed at https://sahildav.github.io/gci18.fossasia.org/

### CODE REVIEW

**Comment by saarthakchats** [comment]
It would be better if Jekyll was used, I think. Look at #13

**Comment by STEdirisuriya** [comment]
Yes. Jekyll made it easier. You can add mentors into that YML after the merge my PR.

**Comment by sahildav** [comment]
@saarthakchats I have Jekyll . This is new PR . Please check the commits  present .

**Comment by sk9331657** [comment]
@sahildav  LGTM !!  @akshatnitd Can this be merged ??

**Comment by saarthakchats** [comment]
@sahildav 
Oh cool, my bad 👍

**Comment by sk9331657** [comment]
@abishekvashok  Can this be merged??

**Comment by sk9331657** [comment]
@mariobehling  Shall I merge this ??

**Comment by YashKumarVerma** [comment]
Change the name of the pull request

**Comment by sahildav** [comment]
@YashKumarVerma  It is fine now ??

**Comment by YashKumarVerma** [comment]
@sahildav Name of the commit needs edit too i guess. 

- **Adding GCI Mentor in Mentor Section** can be changed to ***Adding GCI Mentor Section***

**Comment by YashKumarVerma** [comment]
But the pr adds a mentor section. You can submit this pr for the task even if the name is different.

But since it adds the section, I'll recommended to name it that way

**Comment by sahildav** [comment]
@YashKumarVerma  Is this fine now??

**Comment by sahildav** [comment]
@fragm3  Please review .

**Comment by sahildav** [comment]
@fragm3 Done required changes . Please approve.

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,3 @@
+_site/
+.sass-cache/
+./Gemfile.lock
```
Comment (abishekvashok) [inline .gitignore]: Remove the ./ So that you won't commit the Gemfile.lock

Comment (sahildav) [reply .gitignore]: OK Sir .

Comment (sahildav) [reply .gitignore]: Done @abishekvashok  . Please review .

File: index.html [hunk 2]
```diff
@@ -926,20 +928,49 @@ <h3>Design that looks less 'theme' and more you.</h3>
...
+                                                    {% endif %}
+                                                    {% if mentor.blog %}
+                                                    <a href="https://{{ mentor.blog }}" class="icon-a">
+                                                        <i class="icon-i fa fa-rss fa-2x" aria-hidden="true"></i>
+                                                    </a>
+                                                    {% endif %}
+                                                    {% if mentor.facebook %}
+                                                    <a href="https://facebook.com/{{ mentor.facebook }}" class="icon-a">
+                                                        <i class="icon-i fa fa-facebook fa-2x" aria-hidden="true"></i>
+                                                    </a>
```
[the remark this answers is not in the document]
Comment (sahildav) [reply index.html]: @abishekvashok  EndIf already present.

**Comment by sahildav** [comment]
@YashKumarVerma  Done changes.

File: index.html [hunk 3]
```diff
@@ -926,20 +928,48 @@ <h3>Design that looks less 'theme' and more you.</h3>
...
+                                        <div class="card single-mentor">
+                                            {% if mentor.image %}
+                                                    <img src="img/mentors/{{mentor.image}}" alt="{{ mentor.name }}">
+                                            {% else %}
+                                                <img src="img/mentors/member.png" alt="{{ mentor.name }}">
+                                            {% endif %}
+                                            <br>
+                                            <p class="person-name">{{ mentor.name }}</p><br>
+                                            <ul >
+                                                {% if mentor.github %}
```
Comment (abishekvashok) [inline index.html]: Please properly intend it

File: index.html (hunk 2 above)
[the remark this answers is not in the document]
Comment (abishekvashok) [reply index.html]: Intend the endif then.

File: index.html (hunk 3 above)
Comment (sahildav) [reply index.html]: OK

**Comment by sahildav** [comment]
@abishekvashok  Done required changes.

**Comment by sahildav** [comment]
@abishekvashok  @fragm3  @ms10order398  Please approve and merge this .

**Comment by sahildav** [comment]
@abishekvashok Please approve.

**Comment by sahildav** [comment]
@abishekvashok  Ready to go . :smile:

**Comment by sk9331657** [comment]
@abishekvashok  Shall I merge this ??

**Comment by abishekvashok** [comment]
Thank you!

**Comment by abishekvashok** [comment]
@sk9331657 upsss, just saw the message, sorry!

**Pull request merged**